ingest/src/chunkPage.ts

```diff
@@ -0,0 +1,53 @@
+import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
+import GPT3Tokenizer from "gpt3-tokenizer";
+import { EmbeddedContent, Page } from "chat-core";
+
+export type ContentChunk = Omit<EmbeddedContent, "embedding" | "updated">;
+
+export type SomeTokenizer = {
+  encode(text: string): {
+    bpe: number[];
+    text: string[];
+  };
+};
+
+export type ChunkOptions = {
+  chunkSize: number;
+  chunkOverlap: number;
+  tokenizer: SomeTokenizer;
+};
+
+const defaultChunkOptions: ChunkOptions = {
+  chunkSize: 2000,
```

Comment:
why 2000? seems little high

per convo w simone, perhaps 1000? though would be good to get a source on what an appropriate # should be.

Reply:
https://www.pinecone.io/learn/chunking-strategies#:~:text=Start%20by%20exploring%20a%20variety%20of%20chunk%20sizes%2C%20including%20smaller%20chunks%20(e.g.%2C%20128%20or%20256%20tokens)%20for%20capturing%20more%20granular%20semantic%20information%20and%20larger%20chunks%20(e.g.%2C%20512%20or%201024%20tokens)%20for%20retaining%20more%20context.

Reply:
This is just what was in there already

Reply:
"There" being the seed data script